### ISSUE TITLE
Can not reproduce the result

### ISSUE THREAD

**Issue by Arsiuuu** [wgchang/DSBN#11]
Thanks for your inspiring work! I wonder why I cannot reproduce the result of **Office31(A->W) in stage1 without DSBN.** In the paper it's 91.3 while my best acc is 88.68.
This is my command:
**python trainval_multi.py --model-name resnet50 --exp-setting office --sm-loss --adv-loss --source-datasets amazon --target-datasets webcam --batch-size 40 --save-dir output/resnet50_office_stage1 --print-console**
Is there any mistake? Looking forward to you!

**Comment by woozch** [comment]
Thank you for your interest in my paper. It's been a while since I worked on it, so I don't remember much, but it seems like the commands were correctly written. 
Firstly, since the PyTorch version is old, there might be slightly different results if not reverted to the previous version. Even the CUDA version can affect the training. Additionally, it's important to check whether the parameters of the ResNet50 ImageNet pre-trained model provided by PyTorch match those of the previous version. Also, the values reported in the paper are averages from multiple random seeds, so I recommend running them on different random seeds for validation.
